code/modules/surgery/organs/augments_arms.dm

```diff
@@ -559,3 +559,29 @@
 		qdel(src)
 		qdel(I)
 	return ..()
+
+/obj/item/organ/internal/cyberimp/arm/muscle
+	name = "Strong-arm empowered musculature implant"
```

Comment:
```suggestion
	name = "strong-arm empowered musculature implant"
```

Reply:
Why not capitalized as in the designs?

Reply:
Its an item and i compared with the other names in this file